networkx/algorithms/tests/test_walks.py

```diff
@@ -0,0 +1,64 @@
+"""Unit tests for the :mod:`networkx.algorithms.walks` module."""
+
+import pytest
+
+import networkx as nx
+
+pytest.importorskip("numpy")
+pytest.importorskip("scipy")
+
+
+class TestNumberOfWalks:
+    @classmethod
+    def setup_class(cls):
+        cls.G = nx.DiGraph(nx.utils.pairwise(range(3), cyclic=True))
```

Comment:
Just a nit, but I find this incantation more difficult to grok than simply constructing the cycle graph manually:

```suggestion
        cls.G = nx.DiGraph([(0, 1), (1, 2), (2, 0)])
```